client/coral-admin/src/routes/Moderation/components/Moderation.js

```diff
@@ -137,12 +170,14 @@ export default class Moderation extends Component {
         <ModerationMenu
           asset={asset}
           allCount={root.allCount}
+          newCount={root.newCount}
           acceptedCount={root.acceptedCount}
           premodCount={root.premodCount}
           rejectedCount={root.rejectedCount}
-          flaggedCount={root.flaggedCount}
+          reportedCount={root.reportedCount}
           selectSort={this.props.setSortOrder}
           sort={this.props.moderation.sortOrder}
+          premodEnabled={assetId ? isPremod(root.asset.settings.moderation) : isPremod(settings.moderation)}
```

Comment:
Whether or not premod is enabled should be calculated in the `container` and passed down to this components, as this should be purely presentational.